roles/ceph-osd/tasks/main.yml

```diff
@@ -55,7 +55,10 @@
 
 - name: create openstack pool
   ceph_pool:
-    pool_name: "{{ ceph.pool_name }}"
+    pool_name: "{{ item }}"
+  with_items:
+    - "{{ceph.pool_name}}"
+    - "{{ceph.vm_pool_name}}"
```

Comment:
Our style is to have a space around the mustache brackets. `"{{ ceph.pool_name }}`